app/helpers/application_helper.rb

```diff
@@ -30,4 +30,23 @@ def file_size_for(file, decimal_places=2)
   def format_url_safe(text)
     ERB::Util.url_encode(text)
   end
-end
+
+  def any_fields_present(type, total)
+    if type == 'file'
```

Comment:
You could probably make this a bit cleaner by moving the if inside of the times block, then you can just have one times block for both